ui/raidboss/popup-text.ts

```diff
@@ -1194,6 +1212,30 @@ export class PopupText {
     this._addTextFor('info', triggerHelper);
   }
 
+  _onTriggerInternalRumble(triggerHelper: TriggerHelper): void {
+    if (!this.options.RumbleEnabled)
+      return;
+
+    // getGamepads returns a "GamePadList" which isn't iterable.
+    [...navigator.getGamepads()].forEach((gp) => {
+      if (!gp)
+        return;
+
+      // `vibrationActuator` is supported in CEF but is not in the spec yet.
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const gamepad: any = gp;
+
+      // eslint-disable-next-line max-len
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call
+      void gamepad?.vibrationActuator?.playEffect(gamepad.vibrationActuator.type, {
```

Comment:
For multiple triggers in a row, what does this look like? Does it reset the vibration effect and start over? Do they overlap?

Reply:
Also from the MDN documentation:
```
Note: A new call to playEffect() overrides a previous ongoing call.
```

Reply:
For a single response with both alarm and info, only the alarm rumble will play.  For sequential triggers it will override, as valarnin says above.

I think this is probably fine enough.  Half of the point of a rumble (or sound) is to say "hey look listen" and then partially for the severity itself to indicate extra information.  I would argue that multiple sounds playing is also pretty muddy and we could think about avoiding that too as a more general case of behavior, however neither of them of those feels required to fix here and it'd be "good enough" without that.

I can leave a comment about this behavior, for sure.

Reply:
Other options here could be including a timestamp for when the current rumble will end and not starting a new rumble until that rumble is done? But that could create a priority inversion of info -> alarm where the alarm wouldn't play.

Reply:
Reading the Chromium source code it looks like you get some form of a callback, but even then I don't know how much we'd care to figure all of that out; simply looking at if there's a callback wouldn't be enough and we'd have to store and inspect them. I think following a general principle of "Don't call out multiple things at once, and, if you have to, then call them out in ascending priority order" is already somewhat adhered to, so maybe it just ends up being okay.

Reply:
Yeah, `playEffect` returns a promise when it is `'complete'`, but not clear to me if the callback occurs when the entire effect is done or just when it's submitted.  That's an option too.

Unless you feel strongly, I think my inclination here is to do the simplest thing and when we find a bad case, we can think more about how to handle it.